src/platforms/javascript/common/session-replay/understanding-sessions.mdx

```diff
@@ -97,3 +97,83 @@ await replay.flush();
+ * `replaysOnErrorSampleRate` is set to its default value of "1.0" to always record a replay when an error occurs
+ */
+Sentry.init({
+  dsn: "...",
+  replaysSessionSampleRate: 0.1,
+  replaysOnErrorSampleRate: 1.0,
+  integrations: [replay],
+});
+
+// ...
+// After a user has been authenticated, check if they're an employee
+// If they are, we'll call `replay.flush()` to either flush the replay as normal if it's a session-based replay, or if it's buffering due to error sampling, the flush will send the first segment of the replay.
+
+// Since `replaysOnErrorSampleRate` is > 0, we know that a replay has been buffered and `flush()` will flush the contents of that buffer. If only `replaysSessionSampleRate` is > 1, then there is a chance that a replay has not been recording/buffering.
+// In this case, you can check that `replay.getReplayId()` returns a value, if it does, it means replay is active and you can call `replay.flush()`, other call `replay.start()` to start recording a new replay.
+// ...
+
+// Check if user is an employee, if so, always record a replay
+if (loggedInUser.isEmployee) {
+  // You can get a reference to the Sentry Replay integration like so:
+  const replay = Sentry.getCurrentHub().getIntegration(Sentry.Replay);
+  // or alternatively
+  // const replay = Sentry.getCurrentHub().getClient().getIntegrationById('Replay');
+
+  // `replaysOnErrorSampleRate` is > 0, so a replay will always be buffered while it waits for an error to happen.
+  // `flush()` will create a new replay if it is in a buffering state, otherwise will flush the recording segment if a replay is currently in progress.
+  replay.flush();
+}
+```
+
+### Replays on Specific URLs
+
+You can also choose to record a replay every time a user lands on a specified URL. This can help developers debug problematic pages. To do this, use the [Navigation API](https://developer.mozilla.org/en-US/docs/Web/API/Navigation_API), which can listen to navigation events for a specific URL, start the replay recording, and then stop it when the user leaves the URL.
+
+```javascript
+const replay = new Sentry.Replay();
+
+/**
+ * Initialize the Sentry SDK as normal.
+ *
+ * `replaysSessionSampleRate` and `replaysOnErrorSampleRate` are both set to
+ * "0" so that we have manual control over session-based replays
+ */
+Sentry.init({
+  dsn: "...",
+  replaysSessionSampleRate: 0,
+  replaysOnErrorSampleRate: 0,
+  integrations: [replay],
+});
+
+// Listen to navigation events for when users land on our buggy URL, appropriately named: "/buggy-page/"
+// and then start recording a replay.
+navigation.addEventListener("navigate", (event) => {
+  const url = new URL(event.destination.url);
+
+  if (url.pathname.startsWith("/buggy-page/")) {
+    // User navigates to the buggy page, let's start recording
+    replay.start();
+  } else if (window.location.pathname.startsWith("/buggy-page/")) {
+    // Stop replay when the user leaves the buggy page
+    replay.stop();
```

Comment:
I actually would not recommend that - if we started (not buffered!) there is no need to stop, IMHO it is better to just leave it recording in that case (you can disregard what was captured after the user moved away if you want).

Otherwise you need to implement awaiting on the `stop()`, ensuring stop has finished before you call `start` again etc.